### PR TITLE
[FX-2124] Exposes meta fields on Features

### PR DESCRIPTION
**Pull request by dzucconi**
Re: [FX-2124](https://artsyproduct.atlassian.net/browse/FX-2124)

### CODE REVIEW

File: src/schema/v2/Feature/FeatureMeta.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  description: "Meta-tag related fields for Features",
+  fields: () => ({
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ name }) => `${name} | Artsy`,
+    },
+    description: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ description }) => {
+        if (!description) return DEFAULT_META_DESCRIPTION
```
Comment (damassi) [inline src/schema/v2/Feature/FeatureMeta.ts:23]: Any tests?

Comment (dzucconi) [reply src/schema/v2/Feature/FeatureMeta.ts:23]: 🤷

Comment (dzucconi) [reply src/schema/v2/Feature/FeatureMeta.ts:23]: Really though: I can add some — but Features are the only piece actually guarded by a runtype and the corresponding typing information is threaded through here. So you can have a lot of confidence in the typing.

File: src/schema/v2/Feature/FeatureMeta.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { GraphQLNonNull, GraphQLObjectType, GraphQLString } from "graphql"
+import { Gravity } from "types/runtime"
+import { ResolverContext } from "types/graphql"
+import { markdownToText } from "lib/helpers"
+
+const DEFAULT_META_DESCRIPTION =
+  "Artsy is the world’s largest online art marketplace. Browse over 1 million artworks by iconic and emerging artists, presented by 4000+ galleries and top auction houses in over 100 countries."
```
Comment (dzucconi) [inline src/schema/v2/Feature/FeatureMeta.ts:7]: It seems like this is the sort of thing that should probably be a consistent fallback in our meta tags and extracted to some central const accordingly.

File: src/schema/v2/Feature/FeatureMeta.ts (hunk 1 above)
Comment (dzucconi) [reply src/schema/v2/Feature/FeatureMeta.ts:23]: https://github.com/artsy/metaphysics/blob/9e77f3b8ee1edb9fdbd2c8ec486b02b1bdf341b1/src/lib/loaders/loaders_without_authentication/gravity.ts#L32

File: src/schema/v2/Feature/FeatureMeta.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      resolve: ({ description }) => {
+        if (!description) return DEFAULT_META_DESCRIPTION
+
+        return markdownToText(description)
+      },
+    },
+    image: {
+      type: GraphQLString,
+      resolve: ({ image_urls }) => {
+        return image_urls?.large_rectangle
```
Comment (mzikherman) [inline src/schema/v2/Feature/FeatureMeta.ts:31]: Ah yes, `large_rectangle`, the finest shape there is.

File: src/schema/v2/Feature/FeatureMeta.ts (hunk 1 above)
Comment (damassi) [reply src/schema/v2/Feature/FeatureMeta.ts:23]: There's probably a larger debate here around swapping a long-term culture of tests for types that we should raise somewhere, though the angle makes sense.

Comment (dzucconi) [reply src/schema/v2/Feature/FeatureMeta.ts:23]: I suppose my thinking is that tests on GraphQL fields that don't have logic complex enough to extract is a lot like snapshot tests of React components. The value proposition doesn't quite add up to me: keeping a fixture in sync with the type, etc for what amounts to testing a single conditional when invalid data can't even make it to this resolver at runtime.

Though the expectation that any meta description falls back to a logical default is a totally reasonable thing to have in a spec — I'm going to be working on these types a bit coming up so I'll be sure to add some basic coverage.

Comment (damassi) [reply src/schema/v2/Feature/FeatureMeta.ts:23]: ty 👍 It's not that one tests every bit of logic, but that behavior is tested. A single conditional is easy to reason about, but from a public API perspective that's irrelevant. If we have x number of clients expecting a default and someone comes by and changes that, that breaks consumers.

**Pull request merged**